README.rst

```diff
@@ -242,8 +247,16 @@ should have the format ``_matrix._tcp.<yourdomain.com> <ttl> IN SRV 10 0 <port>
 Note that the server hostname cannot be an alias (CNAME record): it has to point
 directly to the server hosting the synapse instance.
 
-You can then configure your homeserver to use ``<yourdomain.com>`` as the domain in
```

Comment:
I kinda feel like it would make more sense to move the whole "setting up federation" section to a separate file, rather than just the well-known info, but up to you.